src/commands/generateFunctions.ts

```diff
@@ -0,0 +1,241 @@
+            cancellable: false
+        },
+        async () => {
+            await fse.writeFile(openAPIFilePath, openAPIDocString);
+        }
+    ).then(async () => {
+        window.showInformationMessage(localize("openAPIDownloaded", `Downloaded OpenAPI Document for API '${node!.apiContract.name} successfully.`));
+    });
+
+    await window.withProgress(
+        {
+            location: ProgressLocation.Notification,
+            title: localize("generateFunctions", `Generating functions for API '${node.apiContract.name}'...`),
+            cancellable: false
+        },
+        async () => {
+            const args: string[] = [];
+            args.push(`--input-file:${openAPIFilePath} --version:3.0.6314`);
+            args.push(`--output-folder:${uris[0].fsPath}`);
+
+            switch (language.label) {
+                case languageTypes.TypeScript:
+                    args.push('--azure-functions-typescript');
+                    args.push('--no-namespace-folders:True');
+                    break;
+                case languageTypes.CSharp:
+                    args.push(`--namespace:${namespace}`);
+                    args.push('--azure-functions-csharp');
+                    break;
+                case languageTypes.Java:
+                    args.push(`--namespace:${namespace}`);
+                    args.push('--azure-functions-java');
+                    break;
+                case languageTypes.Python:
+                    args.push('--azure-functions-python');
+                    args.push('--no-namespace-folders:True');
+                    args.push('--no-async');
+                    break;
+                default:
+                    throw new Error(localize("notSupported", "Not a supported language. We currently support C#, Java, Python, and Typescript"));
+            }
+
+            ext.outputChannel.show();
+            await cpUtils.executeCommand(ext.outputChannel, undefined, 'autorest', ...args);
+            await promptOpenFileFolder(uris[0].fsPath);
+        }
+    ).then(async () => {
+        window.showInformationMessage(localize("openAPIDownloaded", `Generated Azure Functions app for API '${node!.apiContract.name} successfully.`));
+    });
+}
+
+async function askFolder(): Promise<Uri[]> {
+    const openDialogOptions: OpenDialogOptions = {
+        canSelectFiles: false,
+        canSelectFolders: true,
+        canSelectMany: false,
+        openLabel: "Functions Location",
+        filters: {
+            JSON: ["json"]
+        }
+    };
+    const rootPath = workspace.rootPath;
+    if (rootPath) {
+        openDialogOptions.defaultUri = Uri.file(rootPath);
+    }
+    return await ext.ui.showOpenDialog(openDialogOptions);
+}
+
+async function askJavaNamespace(): Promise<string> {
+    const namespacePrompt: string = localize('namespacePrompt', 'Enter Java namespace folder.');
+    const defaultName =  "com.microsoft.azure.stencil";
```

Comment:
Let's set it to -
```suggestion
    const defaultName =  "com.function";
```

Not adding Microsoft or Stencil in the autogenerated code.